### PR TITLE
feat: mailing system added

### LINKED ISSUE #72: feat: automatic mailing system
I can make an automatic mailing system that will send a mail to a new user after signup.. 

Can you assign this issue to me with the gssoc tag?

### PR DESCRIPTION
**Pull request by SINJINI-G**
fixes #72 

- mailing service added
- signup updated

### CODE REVIEW

File: .gitignore
Comment (PiyushKalyanpy) [inline .gitignore:1]: remove this file from pr

File: package-lock.json
Comment (PiyushKalyanpy) [inline package-lock.json:1]: remove this file from PR

File: pages/signup.tsx [hunk 1]
```diff
@@ -50,7 +51,8 @@ const SignUp = () => {
       draggable: true,
       progress: undefined,
       theme: "light",
-      });
+    });
+    await sendWelcomeMail(user.email);
```
Comment (PiyushKalyanpy) [inline pages/signup.tsx:55]: there is no user variable in this file

**Review by PiyushKalyanpy — CHANGES_REQUESTED** [review]
Please make the changes in your PR

File: pages/signup.tsx (hunk 1 above)
Comment (SINJINI-G) [reply pages/signup.tsx:55]: I just checked that user is declared at line 15.

const [user, setUser] = useState({    // Code at line 15

**Comment by SINJINI-G** [comment]
I have deleted the files but still the conflicts are there.. should I take a pull?

**Comment by PiyushKalyanpy** [comment]
You can close this pr if you want and create a new pr

**Comment by SINJINI-G** [comment]
> fixes #72
> 
> * mailing service added
> * signup updated

I have made a new PR. Kindly check.

**Pull request closed without merging**